source/elements/oneMKL/source/domains/spblas/releasematrixhandle.rst

```diff
@@ -26,8 +26,9 @@ before releasing any data in case of USM.
 
    namespace oneapi::mkl::sparse {
 
-      void release_matrix_handle (oneapi::mkl::sparse::matrix_handle_t  handle,
-                                  const std::vector<sycl::event>        &dependencies = {});
+      sycl::event release_matrix_handle (sycl::queue                           &queue,
+                                         oneapi::mkl::sparse::matrix_handle_t  *handle,
```

Comment:
lets name this p_handle instead of just handle :)

Reply:
might be good to do in initializematrixhandle.rst as well

Reply:
done. :)